README.md

```diff
@@ -180,10 +177,50 @@ The Sulu content management system comes with various services and Twig extensio
 rendering complex websites. This functionality is not available when serving the content of the website in a headless 
 way, therefore the SuluHeadlessBundle includes controllers to **provide JSON APIs for accessing these features**.
 
-- Sulu navigation contexts of the application can be retrieved as JSON object via `{host}/api/navigations/{contextKey}`. 
-Similar to the Twig extension, the API respects the following query parameters `depth`, `flat` and `excerpt`.
+ - [Navigation](#navigation)
+ - [Search](#search)
+ - [Snippet areas](#snippet-areas)
+
```

Comment:
```suggestion
```

think i would not maintain a second list here, the sections below should hopefully be enough. one thing less to think about 🙂